domain/schema/controller/sql/0020-macaroon.sql

```diff
@@ -12,3 +12,10 @@ CREATE TABLE bakery_config (
 -- A unique constraint over a constant index ensures only 1 entry matching the
 -- condition can exist.
 CREATE UNIQUE INDEX idx_singleton_bakery_config ON bakery_config ((1));
+
+CREATE TABLE macaroon_root_key (
+    id BLOB PRIMARY KEY,
```

Comment:
`BLOB` primary key?

Reply:
https://github.com/go-macaroon-bakery/macaroon-bakery/blob/f9b21e15a2ed91756aa172972c7178992c7fe6d1/bakery/dbrootkeystore/rootkey.go#L121

Reply:
Also, the interface for `GetKey` accepts a `id []byte` param

Reply:
Yeah and then in the cache it's cast to string, because you can't use `[]byte` as a map key.

Make it `TEXT`, same with `root_key`. I don't believe we'll even need to cast it.

Reply:
Done